LMS.DataAccess/src/Console/UserMenu/PatronMenu/Borrows/Interfaces/BorrowInput.cs

```diff
@@ -0,0 +1,5 @@
+namespace LMS.DataAccess.Console.UserMenu.PatronMenu.Borrows.Interfaces;
+
+public interface BorrowInput{
+    public void BorrowOption();
+}
```

Comment:
This class also only makes the necessary changes

Reply:
what I mean is that as in all classes you should do only the modification of the namespace and by using